monkestation/code/datums/keybinding.dm

```diff
@@ -0,0 +1,14 @@
+/datum/keybinding/human/equip_swap
+	key = "Ctrl+E"
+	name = "equip_swap"
+	full_name = "Quick-Swap Equipment"
+	description = ""
```

Comment:
there should probably be a description for what this does

Reply:
Added